koku/api/report/ocp/query_handler.py

```diff
@@ -366,6 +368,24 @@ def add_current_month_deltas(self, query_data, query_sum):
 
         return query_data
 
+    def add_vm_storage_units(self, query_data):
+        """
+        Recursively traverse vm query_data (which may be a dict or list)
+        and add a `units` field to each dict found under the storage key list.
+        """
+
+        if isinstance(query_data, dict):
+            for key, value in query_data.items():
+                if key == "storage" and isinstance(value, list):
+                    for pvc in value:
+                        if isinstance(pvc, dict):
+                            pvc["units"] = self._mapper.report_type_map.get("storage_usage_units_key")
+                else:
+                    self.add_vm_storage_units(value)
+        elif isinstance(query_data, list):
+            for item in query_data:
+                self.add_vm_storage_units(item)
```

Comment:
Could we replace this by adding `units=Value("GiB-Mo")` to the `JSONObject` [here](https://github.com/project-koku/koku/pull/5515/files#diff-9d4350ba92eef437d8c29f71cf100e6df7e329e9e9dd21d3981d260ac719c47eR830)?